__tests__/pages/resource/id/evaluate.test.tsx

```diff
@@ -348,7 +348,9 @@ describe("Select component, Radio button, and request preview render", () => {
     await act(async () => {
       fireEvent.click(populationRadio);
     });
-    expect(screen.getByRole("searchbox", { name: "Select Patient" })).toBeDisabled();
+
+    //expects a group autoselect component to be in the document
+    expect(screen.getByRole("searchbox", { name: "Select Group" })).toBeInTheDocument;
 
     //expect the request preview to include reportType=population
     expect(
```

Comment:
The Group resource should be in the request preview as the `subject` (i.e. `subject=Group/<Group id>`) if it is populated. Once that functionality gets added to `evaluate.tsx`, we should add another unit test similar to this one that checks that the request preview includes the `subject` field. We can use similar code to the code that is used to test whether the Practitioner field is populated (occurs around line 321).